.classpath

```diff
@@ -6,6 +6,17 @@
 			<attribute name="module" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/5"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.USER_LIBRARY/paintLib">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.USER_LIBRARY/javafx">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="/home/deekshtih/Downloads/json-simple-1.1.1.jar"/>
+	<classpathentry kind="lib" path="/home/deekshtih/Downloads/json-20200518_1.jar"/>
```

Comment:
it has Deekshith's local system path. Needs to be changed. @muhammedyaseen-git @convict-git can we have a directory of all external dependencies in project repo.; then we can change this path to project repo's dependencies directory.

Reply:
resolved